commands/interceptor/scope.go

```diff
@@ -31,7 +31,7 @@ func ParseScope(name string) schema.Scope {
 		ret = schema.ScopeServiceRelation
 	} else if strings.HasPrefix(name, "service_instance_relation") {
 		ret = schema.ScopeServiceInstanceRelation
-	} else if strings.HasPrefix(name, "service_instance") {
+	} else if strings.HasPrefix(name, "service_instance") || strings.HasPrefix(name, "instance_") {
```

Comment:
I think you should remove this prefix mechanism?

Reply:
Ok, I thought your "auto-scope logic" referred this..

Reply:
Let me be more clear. `auto-scope logic` means the scope is based on the input parameter.